packages/synapse-react-client/src/components/DropdownSelect/DropdownSelect.tsx

```diff
@@ -18,8 +26,12 @@ export type DropdownSelectProps = ButtonGroupProps & {
   selectedIndex?: number
   setSelectedIndex?: (selectedIndex: number) => void
   onButtonClick?: (selectedIndex: number) => void
+  buttonText?: string
   variant?: ButtonProps['variant']
   buttonGroupAriaLabel?: string
+  sx?: SxProps
+  anchorRef?: RefObject<HTMLElement>
```

Comment:
I think this would avoid your need to cast further down
```suggestion
  anchorRef?: RefObject<HTMLDivElement>
```